src/modules/getBullsAndCows.js

```
@@ -13,7 +13,38 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const user = String(userInput).split('');
+  const correct = String(numberToGuess).split('');
```

Similar to the previous comment: converting `numberToGuess` to string is fine, but the function currently trusts the caller to provide 4-digit inputs. If you prefer defensive programming, validate length here or document clearly that validation occurs elsewhere. Either approach is acceptable per the task, but tests may be clearer if this function fails fast on invalid input.